src/System.Windows.Extensions/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\Directory.Build.props" />
+    <PropertyGroup>
```

Comment:
Looks like a type-o here.  Property named `PropertyGroup`.

Reply:
@ericstj I am not able to spot the typo here.


I am working on adding tests to this library. i can add the packaging stuff with that PR.

Reply:
I was actually wrong in my statement, it's just indenting that's off.  `<PropertyGroup` and `<Import` should be aligned.

Reply:
sure I will take care of it in the upcoming PR